plugins/colors.js

```diff
@@ -137,15 +137,13 @@ const ruleFunction = primary => {
         ) {
           return
         }
-
         // Property is shortand and value doesn't include color
         if (
-          (/^border(-top|-right|-bottom|-left|-inline|-block)*$/.test(prop) || /^background$/.test(prop)) &&
-          !valueNode.value.toLowerCase().includes('color')
+          (/^border(?:-top|-right|-bottom|-left|-inline|-block)?$/.test(prop) || /^background$/.test(prop)) &&
+          ['color', 'scale'].every(word => !valueNode.value.toLowerCase().includes(word))
```

Comment:
We now only return if neither `color` nor `scale` is in the token, because we want `--display-blue-scale-0` to fail.

If this is risky we could also consider changing the name of the token to `--display-blue-color-0` or `--display-blue-colorScale-0` CC:@langermank